skaha/utils/logs.py

```diff
@@ -51,3 +54,32 @@ def get_logger(
     logger.propagate = False
 
     return logger
+
+
+def stdout(logs: str) -> None:
+    """Prints log messages with colors based on severity.
+
+    Args:
+        logs (str): The log messages as a single string.
+    """
+    console = Console()
+    lines: List[str] = logs.strip().split("\n")
+
+    for line in lines:
+        # Default style is white.
+        style: str = "white"
+
+        # Set style based on log level indicator.
+        if line.startswith("[W"):
+            style = "yellow"
+        elif line.startswith("[I"):
+            style = "green"
+        # Check for error/warning keywords in messages from other apps.
+        if "ERROR" in line:
+            style = "bold red"
+        elif "WARNING" in line and not line.startswith("[W"):
+            style = "orange1"
+
+        # Create a Rich Text object with the chosen style.
+        text = Text(line, style=style)
+        console.print(text)
```

Comment:
This _looks_ like it will print error information in the main output, which means the return status will be 0 when the user may think it should be non-zero.  Is the intent here just to log something useful?

Reply:
So the util logs are only called, when the user calls, 
```python
session.logs(ids=["uuid"], verbose=True)
async_session.logs(ids=["uuid"], verbose=True)
```

The call signature of this function, already states the output is either `None` or a `Dict`
```python
def logs(self, ids: Union[List[str], str], verbose: bool = False) -> Optional[Dict[str, str]]:
 ```
 
So in `session.logs` we print the logs, and then always return a null
```python
if verbose:
    for key, value in results.items():
        log.info("Session ID: %s\n", key)
        logs.stdout(value)
    return None
````